packages/tables/src/styled/StyledRow.ts

```diff
@@ -96,14 +113,19 @@ const colorStyles = (props: IStyledRowProps & ThemeProps<DefaultTheme>) => {
   `;
 };
 
-export const StyledRow = styled(StyledBaseRow).attrs<IStyledRowProps>(props => ({
+const sizeStyles = (props: IStyledRowProps & ThemeProps<DefaultTheme>) => {
+  return css`
+    height: ${getRowHeight(props)};
+  `;
+};
+
+export const StyledRow = styled(StyledBaseRow).attrs<IStyledRowProps>({
```

Comment:
```suggestion
export const StyledRow = styled(StyledBaseRow).attrs({
```

[nit] type should no longer be needed for the `attrs`.